custom_components/tahoma/binary_sensor.py

```diff
@@ -103,15 +103,19 @@ async def async_setup_entry(
     }
 
     for device in coordinator.data.values():
-        for state in device.definition.states:
-            if description := key_supported_states.get(state.qualified_name):
-                entities.append(
-                    OverkizBinarySensor(
-                        device.device_url,
-                        coordinator,
-                        description,
+        if (
+            device.widget not in IGNORED_OVERKIZ_DEVICES
```

Comment:
Why ignored here?

Reply:
Otherwise, there will be new devices created for the Pod and ProtocolGateway, since they have those identify commands as well.

```python3
IGNORED_OVERKIZ_DEVICES = [
    UIClass.PROTOCOL_GATEWAY,
    UIClass.POD,
]
```